docs/source/developer/contributing.rst

```diff
@@ -930,36 +930,45 @@ User Interface Naming Conventions
 Documents, Files, and Activities
 """"""""""""""""""""""""""""""""
 
-Files are referred to as either files or documents, depending on the
-context.
+Refer to files as either files or documents, depending on the context.
 
-Documents are more human centered. If human viewing, interpretation,
-interaction is an important part of the experience, it is a document in
-that context. For example, notebooks and markdown files will often be
-referring to as documents unless referring to the file-ness aspect of it
+*Documents* are more human centered. If human viewing, interpretation,
+or interaction is an important part of the experience, use the term 
+"document". For example, notebooks and Markdown files will often be
+referred to as documents except in the context of a file system 
 (e.g., the notebook filename).
 
-Files are used in a less human-focused context. For example, we refer to
-files in relation to a file system or file name.
+Use the term *files* in a less human-focused context. For example,
+refer to files in relation to a file system or file name.
 
-Activities can be either a document or another UI panel that is not file
-backed, such as terminals, consoles or the inspector. An open document
-or file is an activity in that it is represented by a panel that you can
-interact with.
+*Activities* are either an opened document or another UI panel that is 
+not related to a file, such as terminals, consoles or the inspector.
+
+Notebook Cells
+""""""""""""""
+
+A notebook contains *cells*, each of which have *input* and one or more 
+*outputs*. When the user runs a cell, the kernel reads and executes the
+input and generates outputs. The notebook then displays the cell's output.
+The term *output* describes one of possibly multiple results of running a
+cell. The term "all output" is ambiguous; it could refer either to all
```

Comment:
Is there any reference to "all output" anywhere else in this doc? 

It seems like there is a recommendation to use "cell output" or "all cell outputs", so referring to "all outputs" seems redundant.

Reply:
This comment is making me want to use `Outputs of All Cells` instead: https://github.com/jupyterlab/jupyterlab/issues/11878#issuecomment-1018817833